src/BuildScriptGeneratorCli/Commands/InstallRunTimeCommand.cs

```diff
@@ -0,0 +1,71 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using System;
+using System.IO;
+using McMaster.Extensions.CommandLineUtils;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Oryx.BuildScriptGenerator;
+using Microsoft.Oryx.Common;
+
+namespace Microsoft.Oryx.BuildScriptGeneratorCli
+{
+    [Command(InstallRunTimeCommand.Name, Description = "Install the required runtime components for a platform.")]
+    internal class InstallRunTimeCommand : CommandBase
+    {
+        public const string Name = "install-runtime";
+
+        [Option(
+            OptionTemplates.Platform,
+            CommandOptionType.SingleValue,
+            Description = "The name of the platform for which the runtime components should be installed.")]
+        public string Platform { get; set; }
+
+        [Option(
+            OptionTemplates.PlatformVersion,
+            CommandOptionType.SingleValue,
+            Description = "The version of the platform for which the runtime components should be installed.")]
+        public string PlatformVersion { get; set; }
+
+        [Option(
+            "--installation-prefix <prefix>",
+            CommandOptionType.SingleValue,
+            Description = "The prefix into which the components will be installed.")]
```

Comment:
Can you give an example of what this looks like? and maybe add it in the description too?

Reply:
it expects a path, and follows the `./configure` paradigm - i'll change the option name and its description

Reply:
how is this path used? is it like a base path where the runtime bits are installed to like a `opt/dotnet` for example if we are installing the .NET Core 2.1 runtime?